src/app/room/components/room.component.ts

```diff
@@ -42,13 +42,16 @@ export class RoomComponent implements OnInit {
 
   ngOnInit(): void {
     this.loadingStatus.isLoading = true;
-    this.currentPlayer$.pipe(take(1)).subscribe(player => {
-      if (!player.id) {
-        this.playerCreationService.createPlayer(this.route.snapshot.params[constants.routeParams.id]).subscribe(_ => this.loadingStatus.isLoading = false);
-      } else {
-        this.loadingStatus.isLoading = false;
-      }
-    });
+    this.currentPlayer$.pipe(
+      take(1),
+      combineLatestWith(this.room$),
+      map(([player, room]) => {
+        if(!player.id && !room?.winner) {
```

Comment:
I think you should rebase and adjust this. This property was renamed to `lastOneActive` (or something) :) 
This will break after the merge otherwise.